README.md

```
@@ -95,6 +95,21 @@ In our release process we rely heavily on visual regression tests using [applito
 
 <!-- </Main description> -->
 
+##  Features
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)